### PR TITLE
Expand testnet validator section in book

### PR DESCRIPTION
**Pull request by CriesofCarrots**
#### Problem
Validators participating in TdS dry run had a lot of questions. Some of these questions have answers in the book; some don't. Either way, we can do a better job answering them.

#### Summary of Changes
Split "Testnet Participation" doc into multiple pages (See `SUMMARY.md`)

@mvines , @garious Hoping for some feedback on organization.
This doesn't contain a lot of new information yet. Troubleshooting and FAQs in particular are waiting on some synthesis of the questions/feedback from the dry run.

Questions:
- Are you on board with this reorg? (It begs for a similar expansion to testnet-replicator, especially when we start giving these more focus)
- The current docs only refer to `testnet.solana.com`. Should we acknowledge our other testnets? This was obviously a bit confusing for TdS participants, especially since all the `solana-wallet` commands default to t.s.c, and don't return their proper TdS balances. How about a "Choosing a Testnet" section, either as its own page or at the start of "Starting a Validator"
- @mvines , I set up a page for staking info. I'll include details about stake warm-up and such. How should we speak to staking for validators connecting to different testnets? (TdS vs. non-TdS)

### CODE REVIEW

**Comment by mvines** [comment]
> Are you on board with this reorg? (It begs for a similar expansion to testnet-replicator, especially when we start giving these more focus)

Yes!   And yep we can later revisit the replicator page to make it follow suit.

> The current docs only refer to testnet.solana.com. Should we acknowledge our other testnets? 

I'm ok with not doing this for now.   I view the TdS instructions as an overlay to this guide.

> especially since all the solana-wallet commands default to t.s.c,

Yeah, I know.  Removing the t.s.c default seems clumsy though.  Perhaps a `solana-wallet config set...` style thing, like `git` does:
```bash
$ solana-wallet config set url https://tds.solana.com:8443
```

> How should we speak to staking for validators connecting to different testnets?

I'm removing the `--stake` argument from `validator.sh` (and generally aiming to delete `validator.sh` as soon as practical).   Instead of `--stake` a validator would just use `solana-keygen` to generate a staking account, and `solana-wallet delegate-stake` to apply it (and it'll attempt to protect you, #5282, from yourself).   But if you prefer, I can fill in that section later as the commits to make this actually happen land.

**Comment by CriesofCarrots** [comment]
@mvines , I'd like to go ahead and merge this soon-ish, so that I can PR updates as functionality changes. Do you have a recommendation for a reviewer?

**Comment by mvines** [comment]
lgtm, r+!

File: book/src/SUMMARY.md [hunk 1]
```diff
@@ -5,9 +5,18 @@
 - [Terminology](terminology.md)
 
 - [Getting Started](getting-started.md)
-  - [Testnet Participation](testnet-participation.md)
-  - [Testnet Replicator](testnet-replicator.md)
-  - [Example: Web Wallet](webwallet.md)
+  - [Running a Validator](running-validator.md)
```
Comment (garious) [inline book/src/SUMMARY.md]: This is too detailed for the "Getting Started" section of the book.  This part of the book aims to be the minimum steps to get to that "Example Client" chapter.  Can you move it further down? Maybe after "Anatomy of a Transaction"?

Comment (CriesofCarrots) [reply book/src/SUMMARY.md]: I'll do that, but it's unfortunate that it ends up below the fold in the navigation menu. (I'd actually like to see some design/usability improvements in the book, including collapsing menu items, which would resolve this particular issue.)

**Pull request merged**